code/modules/client/preferences.dm

```diff
@@ -492,3 +492,130 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 			default_randomization[preference_key] = RANDOM_ENABLED
 
 	return default_randomization
+
+/datum/preferences/proc/handle_client_importing(list/raw_data)
+	if(IsAdminAdvancedProcCall())
+		return
```

Comment:
?

Reply:
@ZephyrTFA You resolved this but code is the same, can you explain why you need this